src/StatusQ/Components/StatusListItem.qml

```diff
@@ -171,7 +171,7 @@ Rectangle {
                 height: visible ? contentHeight : 0
                 wrapMode: Text.WrapAtWordBoundaryOrAnywhere
                 anchors.left: parent.left
-                anchors.right: !statusListItem.titleAsideText && !titleIconsRow.sourceComponent ? parent.right : undefined
+                anchors.right: parent.right
```

Comment:
By doing this fix we are not following the design specs we need. We need that `titleAsideText` and `titleIconsRow` are positioned right next to the text. See the following [PR](https://github.com/status-im/StatusQ/pull/539#discussion_r791840349) where you can see a screenshot of a specific design.

Example to clarify the needed behavior:

- We want this:

![Screenshot 2022-05-04 at 14 08 42](https://user-images.githubusercontent.com/97019400/166678228-1a0a7097-27dd-4537-af54-8172717fbe32.png)

- By the current fix we achive this:

![Screenshot 2022-05-04 at 14 09 24](https://user-images.githubusercontent.com/97019400/166678349-f1c5b252-7153-460d-98a8-f076c0e590e9.png)

Reply:
Good catch @noeliaSD !

Reply:
Oh, its a bit more complicated i've expected :) So if the text is longer than we can place on item, what should we do?
Crop text by clipping or elide text with dots?

Reply:
I'd say yes, but looping in @John-44 to be sure

Reply:
Yes, it is not easy! ;)

What I would suggest is:
- If we configure multiline, it just continues the writting in next lines.
- If we configure as single line, it writes dots.

WDYT @PascalPrecht ?

Reply:
Sure, that works too, if we can make this configurable, even better

Reply:
![image](https://user-images.githubusercontent.com/2522130/166957401-a7b431b5-64cc-4f0c-a4c2-544ca609e9e7.png)

Reply:
![image](https://user-images.githubusercontent.com/2522130/167150168-4da737db-11b9-4c10-87f9-16fb8f0cdd28.png)

Reply:
I've removed multiline property and added tooltip